libraries/AP_Arming/AP_Arming.h

```diff
@@ -141,6 +141,7 @@ class AP_Arming {
     AP_Float                accel_error_threshold;
     AP_Int8                 _rudder_arming;
     AP_Int32                _required_mission_items;
+    AP_Float                _max_arm_dist_m; //maximum allowed distance to first waypoint (m)
```

Comment:
Just a little naming thing, we should rename "_max_arm_dist_m" to "wp_dist_max" I think.